src/features/pages/deriv-ctrader/how-to-start/data.tsx

```diff
@@ -0,0 +1,247 @@
+import React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+import Link from 'features/components/atoms/link'
+import { TradingPlatformStepperViewType } from 'features/components/templates/trade-platform/types'
+
+const how_to_start_images = {
+    step1: (
+        <StaticImage
+            src="../../../../images/common/trading-platforms/ctrader/step-1.png"
+            alt="step"
+            loading="eager"
+            formats={['avif', 'webp', 'auto']}
+            quality={50}
+            objectFit="contain"
+            placeholder="none"
+        />
+    ),
+    step2: (
+        <StaticImage
+            src="../../../../images/common/trading-platforms/ctrader/step-2.png"
+            alt="step"
+            loading="eager"
```

Comment:
Shouldn't it be loading="lazy"?

Reply:
why quality={50}?

Reply:
@NikitK-deriv can you explain why quality is 50?